### ISSUE TITLE
Archiving this repo?

### ISSUE THREAD

**Issue by jakirkham** [conda-forge/libgfortran-feedstock#22]
Would like to suggest we archive this repo. The content here should already be handled by the (new) compilers and associated runtime libraries. It also seems to be confusing users a bit. So it might be worth leaving a note in the README or the repo description explaining this before archiving. Thoughts?

cc @conda-forge/core

**Comment by asmeurer** [comment]
I think archiving disables issues, so if you do so, you should have a pointer for where to go if people have issues. 

Also, unless this has been fixed in conda since I last tried, removing libgfortran breaks gcc because it installs the same package. So to remove it properly you have to also uninstall and reinstall gcc. That would be worth mentioning in the README.

**Comment by jakirkham** [comment]
> I think archiving disables issues, so if you do so, you should have a pointer for where to go if people have issues.

Yep, this seems like good info to have in the README or the description note.

> Also, unless this has been fixed in conda since I last tried, removing libgfortran breaks gcc because it installs the same package.

This is exactly the sort of confusion that we need to avoid. No one should be installing the `gcc` package. It is unmaintained, unused, and ABI incompatible with conda-forge packages. Only the (new) Anaconda compilers should be used.

**Comment by asmeurer** [comment]
As long as the package is there people will install it. What is the correct replacement?

**Comment by isuruf** [comment]
This package is still used in OSX until we switch to a newer gfortran.

**Comment by jakirkham** [comment]
Is that true? I think people are getting [libgfortran-ng]( https://anaconda.org/anaconda/libgfortran-ng/files ). For example [like here]( https://travis-ci.org/conda-forge/openblas-feedstock/jobs/538819743#L432 ).

**Comment by isuruf** [comment]
That's a linux_ppc64le build. Not OSX

**Comment by jakirkham** [comment]
Apologies. Was misled by [the badge]( https://github.com/conda-forge/openblas-feedstock#current-build-status ).

<img width="773" alt="Screen Shot 2019-05-29 at 1 51 26 PM" src="https://user-images.githubusercontent.com/3019665/58579363-dd970a00-8218-11e9-94be-2870aef87e09.png">
